tests/test_cli.py

```diff
@@ -133,19 +134,16 @@ def test_train_command():
         result = runner.invoke(app, [
             "train",
             data_path,
-            model_path,
             label_binarizer_path,
+            model_path,
             "--approach",
             "tfidf-svm",
             "--parameters",
             "{'tfidf__min_df': 1, 'tfidf__stop_words': None}",
-            "--train-info",
```

Comment:
part of the reason you are not catching the problem i mentioned above is you remove that test. btw there is a problem with that test see #154. could we also fix that as part of this PR